compass/ocean/tests/global_ocean/mesh/arm10to60/namelist.split_explicit

```diff
@@ -0,0 +1,9 @@
+config_time_integrator = 'split_explicit'
+config_dt = '00:12:00'
+config_btr_dt = '00:00:30'
```

Comment:
@milenaveneziani looking at our existing cases I see `config_btr_dt = '0000_00:00:24'`, is that a mistake?

Reply:
@darincomeau: as I mentioned above, I don't think this is a mistake. I could double check with Gennaro, who ran the first ARRM60to10 simulation, to see if he remembers something about it.